docs/concepts/vega-protocol.md

```diff
@@ -2,69 +2,273 @@
+* `Volatility (Sigma)` - annualised volatility of the underlying asset:
+  * accepted values: **any strictly non-negative real number**,
+  * suggested value: asset dependent, should be derived from the historical time-series of prices.
+
+<!--### Changing models [WIP]
+
+### Thresholds and rules [WIP]-->
+
+### Propose changes to a market
+Details on proposing changes to market to come.
+
+## Network parameter governance
+There are certain parameters within Vega that influence the behaviour of the system and can be changed by on-chain governance. Vega tokenholders can define the optimal network configuration by creating and voting on network parameter proposals.
+
+Network parameters can only be added and removed with Vega core software releases.
+
+A network parameter is defined by:
+* Name
+* Type
+* Value
+* Constraints
+* Governance update policy
+
+**Read more:** [Guide to submitting a network parameter proposal using the command line](../tutorials/proposals/network-parameter-proposal.md)
+
+### Thresholds for network parameters
+Some network parameters need to be more difficult to change than others. Therefore, the protocol needs to know for each network parameter what governance thresholds apply for ascertaining a proposal's ability to change the parameter's value. Specifically, those thresholds are:
+
+* `MinimumProposalPeriod`
+* `MinimumPreEnactmentPeriod`
+* `MinimumRequiredParticipation` 
+* `MinimumRequiredMajority`
+
+There are groups of network parameters that will use the same values for the thresholds. Importantly, these `minimum` levels are themselves network parameters, and therefore subject to change.
+
+Consider a network parameter that specifies the proportion of fees that goes to validators (`feeAmtValidators`), with change thresholds:
+
+* `MinimumProposalPeriod = 30 days`
+* `MinimumPreEnactmentPeriod = 10 days`
+* `MinimumRequiredParticipation = 60%`
+* `MinimumRequiredMajority = 80%`
+
+A proposal to change the `feeAmtValidators.MinimumProposalPeriod` would need to pass all of the thresholds listed above.
+
+<!--### Threshold and rules [WIP]-->
+  
+## Asset management
+Assets used for trading, paying fees, funding rewards, and providing liquidity need to be deposited using a bridge contract, and can be withdrawn back into an Ethereum wallet if they are not being used for margin or liquidity commitment.
+
 ### Deposits
-For restricted mainnet, the deposits function for Vega is not required. To stake, instead of depositing, tokens must be associated to a Vega key. Tokens used for staking stay in your Ethereum wallet, rather than being held in the ERC20 bridge contract. [Read more about staking VEGA tokens.](./vega-chain#staking-on-vega)
+The first assets that will be available for interacting with markets on Vega will be ERC20 assets. They will need to be deposited into the ERC20 bridge contract. The funds in that smart contract will then be made available to the user's chosen Vega public key.
```

Comment:
Should make clear that each asset (ERC20 token contract) needs to be added the bridge before it will be supported.